specifyweb/frontend/js_src/lib/localization/attachments.ts

```diff
@@ -127,10 +127,6 @@ export const attachmentsText = createDictionary({
     'de-ch': 'Keine Datei',
     'es-es': 'Ningún archivo',
     'fr-fr': 'Pas de fichier',
-    'ru-ru': `
```

Comment:
these translations are in wrong places likely because of merge conflicts when sp7 didn't properly pull the changes from weblate, but kept overwriting what's in weblate